### PR TITLE
base-files: add 'interval' option to netdev LED trigger configuration

### PR DESCRIPTION
**Pull request by RoEdAl**

### CODE REVIEW

**Comment by mkresin** [comment]
Please bump the PKG_RELEASE in package/base-files/Makefile as well.

**Comment by RoEdAl** [comment]
Done.

File: package/base-files/files/bin/config_generate [hunk 1]
```diff
@@ -318,6 +318,7 @@ generate_led() {
 			uci -q batch <<-EOF
 				set system.$cfg.trigger='netdev'
 				set system.$cfg.mode='$mode'
+				set system.$cfg.interval='50'
```
Comment (mkresin) [inline package/base-files/files/bin/config_generate]: Please drop this line. 50 is the default value anyway and I don't see the point in adding a fixed value here.

File: package/base-files/files/etc/init.d/led [hunk 2]
```diff
@@ -72,6 +72,8 @@ load_led() {
 			[ -n "$dev" ] && {
 				echo $dev > /sys/class/leds/${sysfs}/device_name
 				echo $mode > /sys/class/leds/${sysfs}/mode
+				[ -n "$interval" ] && \
```
Comment (mkresin) [inline package/base-files/files/etc/init.d/led]: Please remove the `[ -n "$interval" ] && \` condition. Due to the `config_get interval $1 interval "50"` at the beginning of the the `load_led()` function, `$interval` is set to 50 if not defined,

**Comment by RoEdAl** [comment]
Simplified.

**Comment by mkresin** [comment]
Merged to [my staging tree](https://git.lede-project.org/?p=lede/mkresin/staging.git;a=summary). Thanks!

**Pull request closed without merging**